ncdjango/geoprocessing/params.py

```diff
@@ -402,6 +405,16 @@ def clean(self, value):
             raise ParameterNotValidError
 
         if source == 'service':
+            if '@' in value:
+                value, timestamp = value.split('@', 1)
+
+                try:
+                    service_time = datetime.datetime.utcfromtimestamp(int(timestamp)).replace(tzinfo=pytz.utc)
```

Comment:
From the python (3) docs for utcfromtimestamp: "This may raise OverflowError, if the timestamp is out of the range of values supported by the platform C gmtime() function, and OSError on gmtime() failure. It’s common for this to be restricted to years in 1970 through 2038." Lots of climate datasets go outside that range (on both ends).